manifests/install.pp

```diff
@@ -8,15 +8,43 @@
   user { $go_carbon::user:
     ensure => $go_carbon::ensure,
     gid    => $go_carbon::group,
-    shell  => '/sbin/nologin'
+    shell  => $go_carbon::shell,
   }
 
-  package { $go_carbon::package_name:
-    ensure => $go_carbon::version
+  case $::osfamily {
+    'Debian': {
+      if $go_carbon::download_package {
+        ensure_resource(package, ['wget'])
+        exec { 'download package from release':
+          command => "/usr/bin/wget https://github.com/lomik/go-carbon/releases/download/v${go_carbon::version}/go-carbon_${go_carbon::version}_amd64.deb",
+          cwd     => '/tmp',
```

Comment:
I'm not sure how I feel about this logic that downloads the .rpm to /tmp. In case the rpm gets deleted, puppet will redownload it again even though the package is already installed.
Isn't there a better way ?